src/indicators/drought/chirpsgefs_rainfallforecast.py

```diff
@@ -0,0 +1,227 @@
+"""
+Download CHIRPSGEFS raster data and extract statistics per admin region
+
+[CHIRPS-GEFS](https://chc.ucsb.edu/data/chirps-gefs) is the bias-corrected version of GEFS.
+GEFS is the Global Ensemble Forecast System from NOAA.
+CHIRPS observational data is used for the bias-correction.
+The forecast is published each day for the whole world with a 0.05 resolution.
+It is relatively new, started in 2018, and while it seems a respected source, little research articles exist around it.
+However, GEFS is a well-established source. Forecasts are available starting from 2000
+
+Data limitations:
+- No CHIRPS-GEFS data is available from 01-01-2020 till 05-10-2020.
+This data is available from the [older version of the model](https://data.chc.ucsb.edu/products/EWX/data/forecasts/CHIRPS-GEFS_precip/15day/Africa/precip_mean/),
+but our contact at CHC recommended to not use this
+
+Assumptions
+- The grid cell size is small enough to only look at cells with their centre within the region, not those touching
+"""
+
+import pandas as pd
+import numpy as np
+import geopandas as gpd
+from rasterstats import zonal_stats
+import logging
+from datetime import timedelta
+import rioxarray
+
+from pathlib import Path
+import sys
+import os
+
+path_mod = f"{Path(os.path.dirname(os.path.realpath(__file__))).parents[2]}/"
+sys.path.append(path_mod)
+from src.utils_general.utils import download_ftp
+
+logger = logging.getLogger(__name__)
+
+
+def compute_raster_stats(
```

Comment:
It would help to have the methods in this file in the order that they are used, e.g. first the download method, then `compute_stats_rainyseason` and finally the methods that it calls (which could be made private by adding `_` to their names)